lib/svg-sprite/shape.js

```diff
@@ -712,10 +712,10 @@ SVGShape.prototype.setNamespace = function(ns) {
             });
 
             // Substitute ID references in href attributes
-            select('//@href', this.dom).forEach(href => {
+            select('//@href[not(@xlink:href)]', this.dom).forEach(href => {
```

Comment:
@Kreeg are these changes correct? They are included in a commit that mentions only the tests.

Reply:
@XhmikosR they're correct, but obsolete, I changed these back to previous version.